docs/install-new.md

```diff
@@ -0,0 +1,93 @@
+# Table of Contents
+- [Deploying the infrastructure on Azure](#deploying-the-infrastructure-on-azure)
+- [Setting up Application Gateway Ingress Controller on AKS](#setting-up-application-gateway-ingress-controller-on-aks)
+
+## Deploying the infrastructure on Azure
+
+To create the pre-requisite Azure resources, You can use the following template. It creates:
+1) Azure Virtual Network with 2 subnets.
+2) Azure Application Gateway v2.
+3) Azure Kubernetes Service cluster with required permission to deploy nodes in the Virtual Network.
+4) User Assigned Identity to initialize the aad-pod-identity service and ingress controller.
+5) Set required RBACs.
+
+
+Steps:
+
+1) Create a service principal that will be assigned to aks cluster in the template.
+    ```bash
+    az ad sp create-for-rbac --skip-assignment
+    az ad sp show --id <appId> --query "objectId"
+    ```
+    **Note the appId, password and objectId.**
+
+2) After the above, click to create a custom template deployment. Provide the appId for servicePrincipalClientId, password, objectId in the parameters.
+
+    <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
+        <img src="http://azuredeploy.net/deploybutton.png"/>
+    </a>
+    <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fakshaysngupta%2Fapplication-gateway-kubernetes-ingress%2Fmaster%2Fdeploy%2Fazuredeploy.json" target="_blank">
+        <img src="http://armviz.io/visualizebutton.png"/>
+    </a>
+
+    After the deployment completes, you can look at the Output window for parameters needed in the following steps.
+
+    Example:
+    ![Deployment Output](images/deployment-output.png)
+
+## Setting up Application Gateway Ingress Controller on AKS
+
+Once, we have the created the resources in Azure, we need to deploy following controllers on the cluster:
+1) `aad pod identity` - This controller uses user assigned identity and provide ARM access token to the controller.
+2) `application gateway ingress controller` - This controller communicates ingress related events to the Application Gateway resource.
+
+Steps:
+
+1) To configure kubectl to connect to the deployed Azure Kubernetes Cluster, follow these [instructions](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough#connect-to-the-cluster).
+
```

Comment:
Would be good to add a note, users must ensure that the right resource group and aks cluster name is used to initialize the kubeconfig as part of kubectl setup.   The name "aksCluster' also should match.

```
 az aks get-credentials --resource-group <resource-group>  --name aksCluster
```
For instance, I forgot reinitialize the kubeconfig to the new resource-group I created today and it ended up deploying/updating app-pod-identity in older cluster.